### PR TITLE
Added conditional require statements for blog images on Home and Research pages.

### PR DESCRIPTION
**Pull request by czhou578**
Fixes: #1629 

## Changes

Added conditional require statements for blog images on Home and Research pages. I also added a conditional check for some additional images on the Home page, as demonstrated through the following screenshots.

## Home
image
image
image

## Research

image

## Tests

No new tests were added.

### CODE REVIEW

**Comment by anth-volk** [comment]
Here again, could you instead treat the icon like a thumbnail and place it in the bottom-right corner of a div that fills the space where the image would be?

**Comment by czhou578** [comment]
Is this acceptable?

image
image

File: src/redesign/components/HomePolicyEngineFeatures.jsx [hunk 1]
```diff
@@ -6,7 +6,30 @@ import decileChartScreenshot from "../images/home/decile_chart_screenshot.png";
 import ShowcaseItem from "./ShowcaseItem";
 import useCountryId from "./useCountryId";
 
+function ImageComponent(imagePath) {
+  let imageSrc;
+  try {
+    // Attempt to import the image dynamically
+    imageSrc = require(`../images/home/${imagePath}`).default;
+    console.log(imageSrc);
```
Comment (anth-volk) [inline src/redesign/components/HomePolicyEngineFeatures.jsx:14]: Could you remove this console log?

File: src/redesign/components/HomeBlogPreview.jsx [hunk 2]
```diff
@@ -323,14 +324,25 @@ function BlogTags({ tags }) {
   );
 }
 
+const handleImageLoad = (path) => {
+  // Try to load the image
+  try {
+    return require("../images/posts/" + path);
```
Comment (anth-volk) [inline src/redesign/components/HomeBlogPreview.jsx:330]:
```suggestion
    return require("../../images/posts/" + path);
```

Comment (anth-volk) [reply src/redesign/components/HomeBlogPreview.jsx:330]: We're gradually trying to deprecate the `/redesign` folder, including its images, and the whole thing will be fully removed in an upcoming fix, but for now, this should reference the standard images folder, not the one inside `/redesign`

File: src/redesign/components/ShowcaseItem.jsx [hunk 3]
```diff
@@ -44,16 +46,38 @@ export default function ShowcaseItem({
...
+          />
+        </div>
+      ) : (
+        <img
+          src={image}
+          width={displayCategory === "desktop" ? 400 : "100%"}
+          height="100%"
+          style={{
+            objectFit: "contain",
+            // boxShadow: "0px 0px 10px rgba(0, 0, 0, 0.9)",
```
Comment (anth-volk) [inline src/redesign/components/ShowcaseItem.jsx:76]: Could you remove the commented styling?

**Review by anth-volk — CHANGES_REQUESTED** [review]
Thanks for this, @czhou578! This all looks great, just had some minor changes, and then I think this will be merge-able. Thanks again for your help with this.

**Comment by czhou578** [comment]
Fixed!

**Review by anth-volk — APPROVED** [review]
Thanks for the changes @czhou578! This is perfect and will help increase the stability of the application when errors occur.

**Pull request merged**